iceoryx_posh/source/popo/gateway_generic.cpp

```diff
@@ -28,15 +28,25 @@ GatewayGeneric::GatewayGeneric(const capro::Interfaces f_interface) noexcept
 
 GatewayGeneric::~GatewayGeneric() noexcept
 {
-    if(m_interfaceImpl)
+    if (m_interfaceImpl)
     {
         m_interfaceImpl.destroy();
     }
 }
 
 bool GatewayGeneric::getCaProMessage(CaproMessage& msg) noexcept
```

Comment:
maybe const here?

Reply:
Where? The provided CaProMessage cannot be const as it is set by this method. This is an API like the often used FiFo pop() without optional